packages/dx-react-chart-bootstrap4/src/templates/series/bar.jsx

```diff
@@ -8,10 +7,3 @@ export class Bar extends React.PureComponent {
     );
   }
 }
-
```

Comment:
Why did you remove Bar.propTypes? Can render bar without x,y, width, height props?